WhatsNewInCTSM5.3.md

```diff
@@ -1,53 +1,48 @@
+## Changes to CTSM infrastructure
+ - `manage_externals` removed and replaced by `git-fleximod`
+ - Ability to handle CAM7 in `LND_TUNING_MODE`
 
-Changes to CTSM Answers:
-========================
+## Changes to CTSM answers
 
- Changes to defaults for clm6_0 physics:
+ Changes to defaults for `clm6_0` physics:
   - Urban explicit A/C turned on
-  - Snow thermal conductivity is now Sturm_1997
-  - New IC file for f09 1850
+  - Snow thermal conductivity method is now `Sturm_1997`
+  - New initial conditions file for f09 ("1-degree") 1850, with more in progress
   - New crop calendars
-  - Dust emissions is now Leung_2023
+  - Dust emissions method is now `Leung_2023`
   - Excess ice is turned on
-  - Updates to MEGAN for BVOC's
+  - Updates to MEGAN for BVOCs
   - Updates to BGC fire method
 
  Changes for all physics versions:
-
   - Parameter files updated
   - FATES parameter file updated
   - Glacier region 1 is now undefined
-  - Update in FATES transient Land use
+  - Update in FATES transient land use
   - Pass active glacier (CISM) runoff directly to river model (MOSART)
-  - Add the option for using matrix for Carbon/Nitrogen BGC spinup
+  - Add the option for using Matrix CN method for Carbon/Nitrogen BGC spinup
 
-New surface datasets:
-=====================
+## New surface datasets
 
-- With new surface datasets the following GLC fields have region "1" set to UNSET:
-     glacier_region_behavior, glacier_region_melt_behavior, glacier_region_ice_runoff_behavior
+- With new surface datasets the following GLC fields have region "1" set to UNSET: glacier_region_behavior, glacier_region_melt_behavior, glacier_region_ice_runoff_behavior
 - Updates to allow creating transient landuse timeseries files going back to 1700.
-- Fix an important bug on soil fields that was there since ctsm5.2.0. This results in mksurfdata_esmf now giving identical answers with a change in number of processors, as it should.
-- Add in creation of ne0np4.POLARCAP.ne30x4 surface datasets.
-- Add version to the surface datasets.
-- Remove the --hires_pft option from mksurfdata_esmf as we don't have the datasets for it.
-- Remove VIC fields from surface datasets.
-
-New input datasets to mksurfdata_esmf:
-======================================
-
-- Updates in PFT/LAI/soil-color raw datasets (now from the TRENDY2024 timeseries that ends in 2023), as well as two fire datasets (AG fire, peatland), and the glacier behavior dataset.
+- Fix an important bug on soil fields that was there since `ctsm5.2.0`. This has the side effect of `mksurfdata_esmf` now giving identical answers with a change in number of processors, as it should.
+- Surface datasets now provided for the `ne0np4.POLARCAP.ne30x4` grid.
+- Surface datasets now have their version number embedded to prevent mismatch of surface dataset and CTSM version.
+- Remove the `--hires_pft` option from `mksurfdata_esmf` as we don't have the datasets for it.
+- Remove `VIC` fields from surface datasets.
```

Comment:
What are (were) VIC fields?

Reply:
VIC is an alternative hydrological parameterization that you could turn on, that we are deprecating. It required a few fields on the surface datasets to be used. This added complexity to both surface datasets as well as testing for it. So removing it simplifies a bunch of things all around.

We could look for what fields were added to the surface datasets, but I think just saying they ware removed is sufficient here.